### PR TITLE
ci: helm deployment

### PR DESCRIPTION
**Pull request by eliobischof**

### CODE REVIEW

File: deploy/helm/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+            - start-from-init
+            - --config
+            - /config/zitadel-config-yaml
+            - --config
+            - /.secrets/zitadel-secrets-yaml
+            - --steps
+            - /config/zitadel-config-yaml
+            - --masterkeyFile
+            - /.secrets/zitadel-masterkey
+            - start
```
Comment (fforootd) [inline deploy/helm/templates/deployment.yaml:45]:
```suggestion
```

**Comment by fforootd** [comment]
@eliobischof is this ready to be reviewed, or still a WIP?

**Comment by eliobischof** [comment]
> @eliobischof is this ready to be reviewed, or still a WIP?

I'd like to release it to an indexed public repo for easy usage

File: README.md [hunk 2]
```diff
@@ -20,6 +20,9 @@
         <img src="https://codecov.io/gh/zitadel/zitadel/branch/main/graph/badge.svg" /></a>
     <a href="https://discord.gg/erh5Brh7jE" alt="Discord Chat">
         <img src="https://badgen.net/discord/online-members/erh5Brh7jE" /></a>
+    <a href="https://artifacthub.io/packages/search?repo=zitadel" alt="Discord Chat">
```
Comment (fforootd) [inline README.md:23]:
```suggestion
    <a href="https://artifacthub.io/packages/search?repo=zitadel" alt="ZITADEL Chart">
```

File: docs/docs/guides/installation/helm.mdx [hunk 3]
```diff
@@ -1 +1,10 @@
-Coming soon
\ No newline at end of file
+
+1. [Install Helm](https://helm.sh/docs/intro/install/)
+
+2. Install a ZITADEL Helm release
+```bash
+helm repo add zitadel https://zitadel-charts.storage.googleapis.com/zitadel
```
Comment (fforootd) [inline docs/docs/guides/installation/helm.mdx:6]: Are you sure about this URL?

**Pull request closed without merging**